### DISCUSSION TITLE
Cisco iol

### DISCUSSION THREAD

**Discussion by eayetsi** [srl-labs/containerlab#2260] [Q&A]
I have install Cisco iol images using the documentation. I am running into any issues which I need help. When I run the container, it starts and shows 'running' then within some few minutes, the State changes to 'exit'. Any help?

**Comment by hellt** [comment]
your docker logs may indicate why is it happening

you may want to join the discord to ask generic questions https://discord.com/channels/860500297297821756/865572914346065920

**Reply by eayetsi** [reply]
At the moment, I am not behind my laptop to check the docker logs. I will check and revert.

**Reply by eayetsi** [reply]
> your docker logs may indicate why is it happening
> 
> you may want to join the discord to ask generic questions https://discord.com/channels/860500297297821756/865572914346065920

I have joined the discover but I am not seeing anything. I used the link provided here. 
![image](https://github.com/user-attachments/assets/983bb593-dea5-41a3-a910-9f196da5c6c6)

**Reply by hellt** [reply]
try this link https://discord.gg/vAyddtaEV9

**Reply by eayetsi** [reply]
It is working...Thanks

**Comment by eayetsi** [comment]
Below is the logs I am getiing

Launching IOL with PID
Flushed eth0 addresses
/usr/bin/iouyap: Parsing NETMAP...
/usr/bin/iouyap: --------------
/usr/bin/iouyap: Configuring 513:0/0...
/usr/bin/iouyap: starting foreign listener
/usr/bin/iouyap: --------------
/usr/bin/iouyap: Starting IOU UDS listener
/usr/bin/iouyap: Starting IOU UDP listener
/usr/bin/iouyap: Main thread going to sleep
IOS On Unix - Cisco Systems confidential, internal use only

 IOURC: Could not open iourc file
Missing application ID
